### DISCUSSION TITLE
Auth Systems

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#4794] [Help] [answered]
We setup a budibase app with auth through Azure AD. It seems like users of the app will need budibase accounts to access the apps they have permission to use? Do we need to make them user accounts in the App (default users table etc)?

**Comment by mjashanks** [comment, marked as answer]
Anyone with access on your Azure AD will be able to immediately log into Budibase - there's no need for you to create users. However, they will not have access to any apps by default - and administrator will need to give them access.

We do plan to add the concept of "Workspaces" (user grouping) which should address this, but that would be later this year.

**Reply by CosmasSC128** [reply]
Thank you.

**Comment by CosmasSC128** [comment]
Sorry, did you mean that the permissions are set on Azure AD side, or inside budibase?

**Reply by mjashanks** [reply]
The permissions are still setup inside Budibase. You just don't have to manually add each user.

When we introduce workspaces, we will then also sync with AD groups, which will allow configuration of permissions based on AD groups. Workspaces will be a paid feature.
